### ISSUE TITLE
Support for MyGet Vsix feed

### ISSUE THREAD

**Issue by josesimoes** [badges/shields#1184]
Was trying to use the MyGet badge for the Vsix feed but couldn't make it work.
Could you please clarify if this is supported or not?
Since VS marketplace is now, it seems that it would make sense to have this one too...

**Comment by paulmelnikow** [comment]
I'm not that familiar with these services. Could you provide more context and links?

ping @maartenba @Daniel15

**Comment by josesimoes** [comment]
@paulmelnikow this a service provided by MyGet (for which you already have a few badges applicable to the Nuget feed: version, download count, etc).

You can check the details [here](http://docs.myget.org/docs/walkthrough/getting-started-with-vsix).
(not sure if this is enough....)

(pinging @maartenba @Daniel15 as requested)

**Comment by maartenba** [comment]
It's an Atom-based feed, endpoints documented here: http://docs.myget.org/docs/reference/feed-endpoints#Vsix-compatible_feed_endpoints

Sample feed: https://www.myget.org/F/credentialproviders/vsix

Where `www` is the tenant parameter (can be e.g. `somecompany` and `credentialproviders` is the feed name.

/cc @xavierdecoster

**Comment by paulmelnikow** [comment]
What would these badges show? Put another way, would they be new badges, or modifications to existing badges?

**Comment by josesimoes** [comment]
> What would these badges show?

The same as their equivalents from the VS marketplace: download count or version

>   would they new badges, or modifications to existing badges?

New badges. The current ones (for MyGet) are showing details about the Nuget packages not the VSIX packages.

**Comment by Daniel15** [comment]
Sounds reasonable to me. I guess there's no existing badges that call VSIX APIs?

**Comment by josesimoes** [comment]
@daniel15 Yes there are! Visual Studio market place has been very recently added. :wink:

**Comment by josesimoes** [comment]
Hey guys! Any plans to add this? 😃

**Comment by paulmelnikow** [comment]
Hey @josesimoes, just wanted to put out there that Shields is a community project, and that means you can help make this happen! 👐 We have [a great tutorial](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md) and welcome anyone who would like to dig into this! Our maintainer team is happy to provide guidance on the API design and provide assistance with the implementation. We've had many first-time contributors who don't use JavaScript and have tried to make the first-time contributing experience as straightforward as possible. 

A good place to start on the MyGet badges is by reading this code:

https://github.com/badges/shields/blob/master/services/nuget/nuget-v3-service-family.js

which is used for both NuGet and MyGet.

If you decide to dig in, feel free to post here as you encounter questions!

**Comment by icnocop** [comment]
I'm able to use a [dynamic badge](https://shields.io/#dynamic-badge).

For example:

data type: `xml`
label: `myget`
url: `https://www.myget.org/F/credentialproviders/vsix`
query: `/*[local-name(.)='feed']/*[local-name(.)='entry'][1]/*[local-name(.)='Vsix']/*[local-name(.)='Version']`

Result:

![myget](https://img.shields.io/badge/dynamic/xml?label=myget&query=%2F%2A%5Blocal-name%28.%29%3D%27feed%27%5D%2F%2A%5Blocal-name%28.%29%3D%27entry%27%5D%5B1%5D%2F%2A%5Blocal-name%28.%29%3D%27Vsix%27%5D%2F%2A%5Blocal-name%28.%29%3D%27Version%27%5D&url=https%3A%2F%2Fwww.myget.org%2FF%2Fcredentialproviders%2Fvsix)
